src/main/java/com/jmb/springfactory/service/worklog/WorkLogServiceImpl.java

```diff
@@ -0,0 +1,81 @@
+package com.jmb.springfactory.service.worklog;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.jmb.springfactory.dao.GenericMySQLService;
+import com.jmb.springfactory.dao.worklog.WorkLogMySQLService;
+import com.jmb.springfactory.exceptions.NotFoundException;
+import com.jmb.springfactory.exceptions.ServiceLayerException;
+import com.jmb.springfactory.model.bo.BusinessObjectBase;
+import com.jmb.springfactory.model.dto.TaskDto;
+import com.jmb.springfactory.model.dto.WorkLogDto;
+import com.jmb.springfactory.model.entity.WorkLog;
+import com.jmb.springfactory.service.GenericServiceImpl;
+import com.jmb.springfactory.service.task.TaskService;
+
+@Service
+public class WorkLogServiceImpl extends GenericServiceImpl<WorkLog, WorkLogDto, BusinessObjectBase, Integer> 
+  implements WorkLogService {
+
+  @Autowired
+  private WorkLogMySQLService workLogMySQLService;
+  
+  @Autowired
+  private TaskService taskService;
+
+  @Override
+  public GenericMySQLService<WorkLog, Integer> genericDao() {
+    return workLogMySQLService;
+  }
+
+  @Override
+  public Class<? extends WorkLog> getClazz() {
+    return WorkLog.class;
+  }
+
+  @Override
+  public Class<? extends WorkLogDto> getDtoClazz() {
+    return WorkLogDto.class;
+  }
+
+  @Override
+  public Class<? extends BusinessObjectBase> getBoClazz() {
+    return BusinessObjectBase.class;
+  }
+  
+  @Override
+  public WorkLogDto save(Integer idTask, WorkLogDto workLog) throws NotFoundException, ServiceLayerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)